pkg/ipcache/ipcache.go

```diff
@@ -522,7 +522,11 @@ func (ipc *IPCache) UpsertMetadataBatch(updates ...MU) (revision uint64) {
 	ipc.metadata.Lock()
 	for _, upd := range updates {
 		if !upd.IsCIDR || ipc.metadata.prefixRefCounter.Add(upd.Prefix) {
-			prefixes = append(prefixes, ipc.metadata.upsertLocked(upd.Prefix, upd.Source, upd.Resource, upd.Metadata...)...)
+			resource := upd.Resource
+			if upd.IsCIDR {
+				resource = ipcacheTypes.ResourceID("consolidate-" + upd.Prefix.String())
```

Comment:
Why does the resource owner need to contain the prefix? I don't think this is necessary.

I also prefer it if we used the proper resource owner format that is constructed using `NewResourceID` and has a resource kind like all other owners, for example:

```go
ipcachetypes.NewResourceID(ipcachetypes.ResourceKindDaemon, "", "consolidated-prefix")
```

Like we do for other `ResourceKindDaemon` types.

Reply:
Good point, fixed!